src/sass/login.scss

```diff
@@ -1,5 +1,202 @@
 @import "shared-variables";
 
+span.sidelines {
+  position: relative;
+  z-index: 1;
+  overflow: hidden;
+  text-align: center;
+  width: 100%;
+  display: inline-block;
+
+  font-size: 1.15em;
+  font-weight: bold;
+
+  &:before, &:after {
+    position: absolute;
+    top: 51%;
+    overflow: hidden;
+    width: 50%;
+    height: 2px;
+    content: "\a0";
+    background-color: $color-gray-light-alt;
+  }
+
+  &:before {
+    margin-left: -52%;
+    text-align: right;
+  }
+
+  &:after {
+    margin-left: 2%;
+  }
+}
+
+.verify, .login {
+  button {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    width: 100%;
+    padding: 0.75em;
+    font-size: 16px;
+    margin: 1em 0;
+
+    &.dslogon {
+      background: $color-primary-darker;
+
+      &:hover {
+        background: $color-primary-darkest;
+      }
+    }
+
+    &.idme-create {
+      box-shadow: inset 0 0 0 2px $color-green;
+      color: $color-green;
+      margin-bottom: 0;
+
+      &:hover {
+        box-shadow: inset 0 0 0 2px $color-green-darker;
+        color: $color-green-darker;
+      }
+    }
+
+    img {
+      height: 1em;
+      padding: 0 0.25em;
+    }
+  }
+
+  .container {
+    padding: 2em 0;
+  }
+
+  .logo {
+    text-align: center;
+    border-bottom: solid 2px $color-gray-light-alt;
+    padding: 1em 0;
+
+    a {
+      &:hover {
+        background-color: transparent;
+      }
+      display: inline-block;
+    }
+  }
+
+  .help-info {
+    padding: 2em 0;
+
+    p:first-of-type {
+      margin-top: 0;
+    }
+  }
+}
+
+.verify {
+  button {
+    width: auto;
+
+    &.dslogon {
+      margin-bottom: 0.25em;
+    }
+  }
+
+  @include media-maxwidth($small-screen) {
+    padding: 1em;
```

Comment:
This can be removed - it's giving the verify screen more padding than the sign in screen has, they should be consistent.